internal/budgetadjustments/events/events.go

```diff
@@ -0,0 +1,21 @@
+package events
+
+import (
+	_ "embed"
```

Comment:
redundant embed

Reply:
Yes, I will remove it. BTW this is cool name for a band "Redundant embed" :guitar: